atoma-service/src/handlers/chat_completions.rs

```diff
@@ -1434,10 +1447,10 @@ pub mod utils {
             level = "info",
             endpoint = "handle_non_streaming_response",
             stack_small_id = stack_small_id,
-            estimated_total_compute_units = estimated_total_compute_units,
+            estimated_total_compute_units = estimated_input_tokens + estimated_output_compute_units,
```

Comment:
we have the input tokens and output compute units, we should stick to one term only (tokens or compute units). At this point, I don't mind using tokens